packages/api/cms-api/src/user-permissions/auth/user-permissions.guard.ts

```diff
@@ -25,7 +25,7 @@ export class UserPermissionsGuard implements CanActivate {
         if (!user) return false;
 
         // System user authenticated via basic auth
-        if (user === true) return true;
+        if (user === "static-credentials-basic") return true;
```

Comment:
This makes assumptions about how the strategy is named. Is this a good idea?

Reply:
if we make it configurable, we can't hardcode it here

Reply:
We could also get the permissions and/or scopes for system users via the AccessControlService, which would be even more flexible. What do you think?

Reply:
> Then let's make it configurable

you should mention that also in the migration guide

Reply:
> We could also get the permissions and/or scopes for system users via the AccessControlService, which would be even more flexible. What do you think?

what would be the use case for this?

Reply:
> what would be the use case for this?

I'm thinking about 3rd party requests that should only be allowed for specific endpoints. But maybe that's not a real use case because we need to disable the Comet guards for such endpoints anyway